packages/web/src/types/images.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.png" {
```

Comment:
This should solve the issue of TS complaining about png files, did it not?

Reply:
It looks like it was an issue with duplicated code.